step_04/solution.dart

```diff
@@ -1,6 +1,9 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
+// Hmmm, is this solution working for you in DartPad? I worry I might be running
+// into a bug :/ I cannot use Escape to clear text, and when I hit Control +
+// Enter it simply moves to the next page?
 class LoginPage extends StatefulWidget {
```

Comment:
I tested it now as well and it is working, but maybe the cache/focus problem that is discussed before is happening :(

Reply:
Yah it seems inconsistent. Luckily today everything is going smooth so far!

Reply:
I made couple of tiny improvements to increase the user experience :)